### PR TITLE
[MetaSchedule] Misc improvement of the Measurer

### PR DESCRIPTION
**Pull request by sunggg**
Nice to meet you all. 

This PR contains several changes in meta_scheudle runner and builder.
Please review this and feel free to give any feedback.

Thank you!

### CODE REVIEW

File: python/tvm/meta_schedule/builder/local_builder.py [hunk 1]
```diff
@@ -15,20 +15,36 @@
...
-from tvm.runtime import Module
+from tvm.runtime import Module, NDArray, load_param_dict, save_param_dict
 from tvm.target import Target
 
 from ...contrib.popen_pool import MapResult, PopenPoolExecutor, StatusKind
 from ..utils import cpu_count, get_global_func_with_default_on_worker
 from .builder import BuilderInput, BuilderResult, PyBuilder
 
 
+logger = logging.getLogger(__name__)
```
Comment (junrushao1994) [inline python/tvm/meta_schedule/builder/local_builder.py:33]: Likely we will need to append `# pylint: disable=invalid-name` to this line to suppress the linter

File: python/tvm/meta_schedule/runner/local_runner.py [hunk 2]
```diff
@@ -25,12 +27,14 @@
...
 from .runner import PyRunner, RunnerFuture, RunnerInput, RunnerResult
 from .utils import (
-    T_ARG_INFO_JSON_OBJ_LIST,
     T_ARGUMENT_LIST,
+    T_ARG_INFO_JSON_OBJ_LIST,
     alloc_argument_common,
     run_evaluator_common,
 )
 
+logger = logging.getLogger(__name__)
```
Comment (junrushao1994) [inline python/tvm/meta_schedule/runner/local_runner.py:36]: ditto

File: python/tvm/meta_schedule/runner/rpc_runner.py [hunk 3]
```diff
@@ -31,13 +32,16 @@
...
 from .utils import (
-    T_ARG_INFO_JSON_OBJ_LIST,
     T_ARGUMENT_LIST,
+    T_ARG_INFO_JSON_OBJ_LIST,
     alloc_argument_common,
     run_evaluator_common,
 )
 
 
+logger = logging.getLogger(__name__)
```
Comment (junrushao1994) [inline python/tvm/meta_schedule/runner/rpc_runner.py:42]: ditto

**Review by junrushao1994 — APPROVED** [review]
Overall looks good! @sunggg let's fix the CI :-)

**Comment by junrushao1994** [comment]
CC: @Hzfengsy @yidawang

File: python/tvm/meta_schedule/builder/local_builder.py (hunk 1 above)
Comment (sunggg) [reply python/tvm/meta_schedule/builder/local_builder.py]: Thank you for the feedback! I've just pushed the updated commit.

**Review by comaniac — APPROVED** [review]
LGTM

**Comment by junrushao1994** [comment]
Hey you forgot the fix the two "ditto"s :-)

**Review by Hzfengsy — APPROVED** [review]
LGTM

**Comment by sunggg** [comment]
> 

Aren't there three files in total? 
My new commit contains changes in three files you mentioned.

**Comment by junrushao1994** [comment]
Oh I’m sorry that I misread! The linter error this time is that we forgot to add docs for “params” in c++

File: python/tvm/meta_schedule/builder/local_builder.py (hunk 1 above)
Comment (junrushao1994) [reply python/tvm/meta_schedule/builder/local_builder.py]: Thanks! just click “resolve conversation” yourself if the comment is addressed :-)

**Comment by junrushao1994** [comment]
CC @zxybazh for review

**Review by zxybazh — APPROVED** [review]
LGTM. Thanks @sunggg !

**Pull request merged**